### ISSUE TITLE
Pattern table 1.16.5

### ISSUE THREAD

**Issue by Baj789** [SlimeKnights/TinkersConstruct#4613]
### Minecraft Version

1.16.5

### Forge Version

36.2.2

### Mantle Version

1.7.11

### Tinkers' Construct Version

1.16.5

### Describe your issue

I was attempting to learn how to use the new tinkers, as i havent played it in forever and its clearly changed. But anyway, i was using the pattern table, which is now the part builder and the stencil table combined which is fine, its useful even. But it wont let me craft any of the parts. i tried wooden planks, logs, cobble stone and clay. None of them would craft into any part. i looked up other people using this version of tinkers, but it works fine for them. I am running this all on All the mods 6. 

### Crash Report

n/a

### Reproduced with only Tinkers' Construct?

No

### Other mods required

i havent tried to reproduce this because i dont really know how to just get tinkers

### 

- [X] Confirm you have checked the roadmap, what's new, and the FAQ for your issue
- [X] Confirm you are not using performance enhancers, such as optifine

**Comment by KnightMiner** [comment]
Table works fine. Try reading the book, you likely forgot patterns or something similar. You might also have an out of date build, but I cannot tell because you did not provide a useful version number (obviously you are using a 1.16.5 tinkers on 1.16.5, we have a mod specific version, like 3.1.1)

If you want a reopen, provide more information such as exact version number and screenshots.

**Comment by Baj789** [comment]
Well we figured out its something to do with the host. It was a Lan server and the host could work it fine, but i couldnt. And if i loaded into a single player world i could do it just fine. I'm using 3.1.2.265. I dont know what all the different numbers mean because when i looked up the issue on any other platform they didnt use the 3.1.2, just 1.16.5
Also i did read the book, all of the books. I actually noticed that materials and you isnt the same as other people playing on the same version. But its whatever.

Anyway, its a host issue, and we got around it by oping myself and the other players
